buildenv/jenkins/jck_sync

```diff
@@ -0,0 +1,133 @@
+                try {
+                    def gitConfig = scm.getUserRemoteConfigs().get(0)
+                    // Adopt windows machines require env here https://github.com/adoptium/aqa-tests/issues/1803
+                    ref_cache = "${env.HOME}/openjdk_cache"
+                    timeout(time: 1, unit: 'HOURS') {
+                        forceCleanWS()
+                    }
+                    checkout scm: [$class: 'GitSCM',
+                    branches: [[name: "${scm.branches[0].name}"]],
+                    extensions: [
+                        [$class: 'CleanBeforeCheckout'],
+                        [$class: 'CloneOption', reference: ref_cache],
+                        [$class: 'RelativeTargetDirectory', relativeTargetDir: 'aqa-tests']],
+                    userRemoteConfigs: [[url: "${gitConfig.getUrl()}"]]
+                    ]
+
+                    syncJckMaterial()
+
+                    } catch (Exception e) {
+                        println("Exception: " + e.toString())
+                        // build result may not be updated correctly at the moment (see https://issues.jenkins.io/browse/JENKINS-56402)
+                        // if there is an exception, set currentBuild.result to ABORTED/FAILURE
+                        if (e.toString().contains("FlowInterruptedException")) {
+                            currentBuild.result = 'ABORTED'
+                        } else {
+                            currentBuild.result = 'FAILURE'
+                        }
+
+                    }
+                }
+    }
+}
+
+def getGitRepoBranch(defaultOwnerBranch, repo) {
+	String[] actualRepoBranch = defaultOwnerBranch.split(":")
+	String owner = actualRepoBranch[0].trim()
+	String repoURL = "https://github.com/${owner}/${repo}.git"
+	actualRepoBranch[0] = repoURL
+	actualRepoBranch[1] = actualRepoBranch[1].trim()
+	return actualRepoBranch
+}
+
+def forceCleanWS() {
+    try {
+        cleanWs disableDeferredWipeout: true, deleteDirs: true
+    } catch (Exception e) {
+        echo 'Exception: ' + e.toString()
+        //cleanWs has issue to delete workspace that contains non-ASCII filename in TKG output https://issues.jenkins.io/browse/JENKINS-33478
+        //cannot delete workspace directly. Otherwise, Jenkins job will abort due to missing workspace
+        sh "rm -rf ${env.WORKSPACE}/aqa-tests/TKG"
+        // call cleanWs() again
+        cleanWs disableDeferredWipeout: true, deleteDirs: true
+    }
+}
+
+def syncJckMaterial(){
+        echo "Starting script to sync JCK materials ....."
+        def JCK_VERSION = params.JDK_VERSION ? "-j ${params.JDK_VERSION}": ""
+        def GIT_DEV_BRANCH = params.JCK_GIT_BRANCH ? "-gb ${params.JCK_GIT_BRANCH}": ""
+        def JAVA_HOME = params.JAVA_HOME ? "-java ${params.JAVA_HOME}": ""
+        def ARTIFACTORY_URL = params.ARTIFACTORY_URL ? "-au ${params.ARTIFACTORY_URL}": ""
+        def JCK_GIT_REPO = params.JCK_GIT_REPO ? "-repo ${params.JCK_GIT_REPO}": ""
```

Comment:
We should unify the names. Please use `JCK_GIT_REPO` and `JCK_GIT_BRANCH` in the job, pipeline, and jckupdater.sh.

Reply:
changes are made to have unify names in job pipeline and jckupdater.sh